CONTRIBUTING.rst

```diff
@@ -223,9 +223,9 @@ details on these requirements.
 Type annotations
 ----------------
 
-The ``cloud-init`` codebase uses Python's annotation support for storing
+The ``cloud-init`` code base uses Python's annotation support for storing
 ``type`` annotations in the style specified by `PEP-484`_ and `PEP-526`_.
-Their use in the codebase is encouraged.
+Their use in the code base is encouraged.
```

Comment:
We typically keep codebase as a single word as well. Although it's a compound noun it's been in common use for so long that it's practically become a noun at this point. I think in this case it's better to keep it as a single word.